src/plugins/irc-events/away.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const _ = require("lodash");
+const Msg = require("../../models/msg");
+
+module.exports = function(irc, network) {
+	const client = this;
+	irc.on("away", (data) => {
+		let away = null;
+
+		if (data.message) {
```

Comment:
If `away.message` is an empty string here and you do want undefined:
```
const away = data.message || undefined;
```
otherwise just doing `const away = data.message;` is enough.